modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -2597,6 +2597,42 @@ TEST_P(Test_ONNX_layers, where_node)
     testONNXModels("where_layer");
 }
 
+#if CV_TRY_AVX2 || CV_TRY_AVX || CV_TRY_NEON
+TEST_P(Test_ONNX_layers, Conformance_Gemm_all_attributes) {
```

Comment:
I would like suggest to create a common class with `SetUp` and skip the tests if no CPU feature detected in runtime by `cv::checkHardwareSupport`td